src/async-components/views/dialogs/eventindex/ManageEventIndexDialog.tsx

```diff
@@ -26,14 +25,23 @@ import {formatBytes, formatCountLong} from "../../../../utils/FormattingUtils";
 import EventIndexPeg from "../../../../indexing/EventIndexPeg";
 import {SettingLevel} from "../../../../settings/SettingLevel";
 
+interface IProps {
+    onFinished: (boolean) => {},
```

Comment:
according to EventIndexPanel this method is void and does not care for a boolean

Reply:
Well, the `boolean` is the usual one for modals, which is where this function ends up going. It's true the `EventIndexPanel` caller doesn't currently use that value. Fixed the return syntax.